### PR TITLE
Landing Screen

### PR DESCRIPTION
**Pull request by vstarzf**
Implements the Landing Screen (Step 1 in User Onboarding flow)

- Also adds PNGs from Figma for the Uplift & Appdev logos present on the screen

Screenshot from Emulator:
image

Figma:
image

### CODE REVIEW

**Comment by thisjustin123** [comment]
You should aim to hide the purple status bar up top.

Perhaps check this link: https://stackoverflow.com/questions/69688138/how-to-hide-navigationbar-and-statusbar-in-jetpack-compose

File: app/src/main/java/com/cornellappdev/uplift/ui/screens/LandingScreen.kt [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        Spacer(modifier = Modifier.height(387.dp));
+
+        Image(
+            painter = painterResource(id = R.drawable.ic_appdev_logo),
+            contentDescription = "Cornell AppDev logo",
+            modifier = Modifier
+                .height(26.dp)
+                .width(158.dp)
+        )
```
Comment (thisjustin123) [inline app/src/main/java/com/cornellappdev/uplift/ui/screens/LandingScreen.kt]: To make the bottom part equidistant from the bottom on all screen sizes, convert the outer container to a box, and then make the uplift logo padded relative to the top, and the cornell appdev logo padded relative to the bottom.

**Review by thisjustin123 — DISMISSED** [review]
Tweak plz!

**Comment by vstarzf** [comment]
> You should aim to hide the purple status bar up top.
> 
> Perhaps check this link: https://stackoverflow.com/questions/69688138/how-to-hide-navigationbar-and-statusbar-in-jetpack-compose

fixed:

image

File: app/src/main/java/com/cornellappdev/uplift/ui/screens/LandingScreen.kt (hunk 1 above)
Comment (vstarzf) [reply app/src/main/java/com/cornellappdev/uplift/ui/screens/LandingScreen.kt]: done

**Review by thisjustin123 — APPROVED** [review]
Noice 😎

**Pull request merged**